app/background/find_featured_posts.rb

```diff
@@ -0,0 +1,14 @@
+class FindFeaturedPosts
+  include Sidekiq::Worker
+
+  def perform
+    Post.all.each do |post|
+      likes = post.likes.where('created_at > ?', 5.days.ago)
```

Comment:
From what time is the 5.days.ago?

Reply:
1) You are grabing all the post
2) For each post you make a query of the likes
3) Of thos likes you are doing another query

Is this N+1 problem? 1 query for loading the posts and n querry for loading the likes. Maybe a cooler way is to add `include(:likes)`